### PR TITLE
fix(babel-preset-gatsby-package): Add corejs to config

### LINKED ISSUE #17706: babel-preset-gatsby-package doesnt specify corejs in babel preset
## Description

While trying to develop a Gatsby plugin I get the following errors

```
WARNING: We noticed you're using the `useBuiltIns` option without declaring a core-js version.
```

This is because `babel-preset-gatsby-package` does not specify corejs in the preset

https://github.com/gatsbyjs/gatsby/blob/c5cc361d26b2e9e0b31ec68e9f040d7d97c4572f/packages/babel-preset-gatsby-package/index.js

But `babel-preset-gatsby` does.

https://github.com/gatsbyjs/gatsby/blob/c5cc361d26b2e9e0b31ec68e9f040d7d97c4572f/packages/babel-preset-gatsby/src/index.js

### PR DESCRIPTION
**Pull request by gonzarodriguezt**
## Description

This is a warning in development about a missing `corejs` config in babel, the issue below tracks down the source of the problem, and this fix quiets it down, thanks!

Edit: I specified the version 3 as it seems to be the recommended version on the docs [(here)](https://github.com/zloirock/core-js#babelpreset-env)

## Related Issues

Fixed #17706

### CODE REVIEW

File: packages/babel-preset-gatsby-package/index.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ function preset(context, options = {}) {
         r(`@babel/preset-env`),
         Object.assign(
           {
+            corejs: 3,
```
Comment (wardpeet) [inline packages/babel-preset-gatsby-package/index.js]:
```suggestion
            corejs: 2,
```

**Comment by gonzarodriguezt** [comment]
Oh sorry, just committed the changes, thanks for explaining it!

File: packages/babel-preset-gatsby-package/package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "@babel/preset-env": "^7.6.0",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
-    "babel-plugin-dynamic-import-node": "^1.2.0"
+    "babel-plugin-dynamic-import-node": "^1.2.0",
+    "core-js": "2"
```
Comment (sidharthachatterjee) [inline packages/babel-preset-gatsby-package/package.json]:
```suggestion
    "core-js": "^2.6.9"
```

Comment (gonzarodriguezt) [reply packages/babel-preset-gatsby-package/package.json]: Done!

**Comment by gonzarodriguezt** [comment]
Thanks to you guys! Super glad to contribute 😄

**Comment by baerrach** [comment]
Cheers!
What does the release process look like?
That is, when can I start using it :P

**Comment by pieh** [comment]
@baerrach sorry for late reply - I'm currently publishing, so it should be out in few minutes, will post new comment when it's done

**Comment by pieh** [comment]
`babel-preset-gatsby-package@0.2.5` is published!

**Comment by baerrach** [comment]
Look at that lack of warning!

Thanks.

**Pull request merged**